### PR TITLE
AGENT-992: installer presubmit job to test control plane with 4 and 5 replicas

### PR DESCRIPTION
**Pull request by rwsu**
The job's name is e2e-agent-5control-ipv4.

Requires: https://github.com/openshift/assisted-service/pull/6917

### CODE REVIEW

File: ci-operator/config/openshift/installer/openshift-installer-master.yaml [hunk 1]
```diff
@@ -1011,6 +1011,17 @@ tests:
     cluster_profile: equinix-ocp-metal
     workflow: agent-e2e-ha-dualstack
   timeout: 6h0m0s
+- as: e2e-agent-5control-ipv4
+  cluster: build05
```
Comment (andfasano) [inline ci-operator/config/openshift/installer/openshift-installer-master.yaml:1015]: shouldn't be optional this job?

Comment (rwsu) [reply ci-operator/config/openshift/installer/openshift-installer-master.yaml:1015]: Yes, it should be optional. Thanks for catching it.

**Comment by rwsu** [comment]
I updated the PR to add a job for 4 replicas.

**Pull request merged**